modules/standard/IO.chpl

```diff
@@ -11806,7 +11806,13 @@ proc readf(fmt:string):bool throws {
    :throws UnexpectedEofError: Thrown if EOF encountered skipping field.
    :throws SystemError: Thrown if the field could not be skipped.
  */
+@deprecated("skipField is deprecated, please use JsonDeserializer instead.")
 proc fileReader.skipField() throws {
+  this._skipField();
+}
+
+@chpldoc.nodoc
+proc fileReader._skipField() throws {
```

Comment:
Will this hidden version be going away in favor of using `JsonDeserializer` even in our modules at some point?

Reply:
The couple of uses in ``ChapelIO`` are used to implement the old readThis/writeThis functionality, and will be removed eventually as well, so we won't have to replace it with anything.

Reply:
Ok, sounds good